tls/tls.go

```diff
@@ -17,8 +17,23 @@ import (
 	"golang.zx2c4.com/wireguard/wgctrl/wgtypes"
 )
 
-// CERTTIFICAT_VALIDITY duration of certificate validity in days
-const CERTIFICATE_VALIDITY = 365
+const (
```

Comment:
some unused func in this package

Reply:
We saw the unused funcs but decided to leave them in in case you saw a need for them, but can take them out